Utilities/ShimUtils/README.md

```diff
@@ -0,0 +1,105 @@
+## OpenCore + OpenLinuxBoot + Secure Boot
+
+If you want to use OpenCore + OpenLinuxBoot + Secure Boot it is possible to self-sign
+everything, including any new Linux kernels after updates. This is possible since most
+standard distros leave at least the previous kernel bootable (and OpenLinuxBoot exposes
+this, via the Auxiliary menu), so you can boot into the old kernel, then own-sign the
+new kernel.
+
+More convenient may be to trust the signing keys of the specific distros which you
+want to boot, which are bundled into the `shimx64.efi` file installed with each distro.
+You can extract these with `shim-to-cert.tool` distributed with OpenCore, then install
+them in your system Secure Boot `db` variable. Best practice would be to install the deny
+list (`vendor.dbx`) from `shimx64.efi`, if any, into your system `dbx` variable, as well.
+(Otherwise you are ignoring any recovations which the vendor has made.)
+
+Recently, Shim has added SBAT support, as a more efficient way to revoke unsafe
+binaries. Unfortunately, the SBAT enforcement code is part of Shim, it is not
```

Comment:
`and it is not`?

Reply:
Gone for 'and is not'

Reply:
I changed some but left some others of these comments - tyvm for the review.